app/controllers/newflow/educator_signup_controller.rb

```diff
@@ -161,7 +161,7 @@ def educator_cs_verification_form
 
     def educator_pending_cs_verification
       security_log(:user_viewed_signup_form, form_name: action_name)
-      @email_address = current_user.email_addresses.first&.value
+      @email_address = current_user.email_addresses.last&.value
```

Comment:
what does this accomplish vs `first`?

Reply:
Previously, users during signup could only ever have one email address, so the `first` or `last` email address was the same. However, now, users may have an email address that they signed up with (which could have come from Gsuite or facebook, if not typed in) and then a "school-provided" email address which they want us to use to verify their identity as educators. Therefore, I want to use *that* email address — which is the **last** added to their account.